consensus/core/src/commit_syncer.rs

```diff
@@ -647,7 +643,22 @@ impl<C: NetworkClient> CommitSyncer<C> {
             sleep(forward_drift).await;
         }
 
-        Ok((commits, fetched_blocks))
+        // 9. Now create the Certified commits by assigning the blocks to each commit. The last commit will also bear the votes with it.
```

Comment:
nit: "9. Now create the Certified commits by assigning the blocks to each commit and retaining the commit votes history.